src/Microsoft.AspNetCore.Razor/Compilation/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -200,5 +202,23 @@ public IEnumerable<TagHelperRequiredAttributeDescriptor> RequiredAttributes
         /// tag helper.
         /// </summary>
         public TagHelperDesignTimeDescriptor DesignTimeDescriptor { get; set; }
+
+        public Dictionary<string, string> PropertyBag
```

Comment:
- Expose the interface `IDictionary<string, string>`.
- Gain pranav points: `public Dictionary<string, string> PropertyBag => _propertyBag`
- Add doc comments.